src/components/ProductTour/ProductTour.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import styles from './ProductTour.scss';
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+    /** Should the tour be shown */
+    isOpen: boolean;
+    /** An array of ReactElement where each element is one slide */
+    children: React.ReactElement[];
+    /** The label for the checkbox. If undefined or empty, checkbox will not be shown */
+    checkboxLabel?: string;
+    /** Cancel button label - required if there are is then 1 slide */
+    cancelLabel?: string;
+    /** Continue button label - required if there are is then 1 slide */
+    continueLabel?: string;
+    /** Finish button label */
+    finishLabel: string;
+    /** A callback if the user clicked the Cancel button without finishing the tour */
+    onCancel?: (isChecked?: boolean) => void;
```

Comment:
What do you think about passing the last viewed step in the second argument? Actually, I don't know why we need this information :) Maybe we can use that information for analytics or we can start  `ProductTour` from that step when the user will back the next time.

Reply:
I think it is a good idea, but I would add it when we have a usecase for it. Then we are sure we are doing what we need. E.g. if you want to follow on analytics, you might want to log all events of clicking on any of the buttons. In that case we should add an `onContinue`, and for the dots we could have an `onDotClick` params. But in the usecase you have, we would need to change the args on the current callback. I think they are all valid use cases, let's see which one we will need once there are requirements for it.